src/picongpu/include/traits/attribute/GetCharge.hpp

```diff
@@ -100,8 +100,9 @@ HDINLINE float_X getCharge(const float_X weighting, const T_Particle& particle)
     typedef T_Particle ParticleType;
     typedef typename PMacc::traits::HasIdentifier<ParticleType, boundElectrons>::type hasBoundElectrons;
```

Comment:
we add our comment, the handling of "atomic numbers XOR chargeRatio" and asserts in a follow-up PR